charts/kagenti/values.yaml

```diff
@@ -40,7 +41,7 @@ common:
 #  Agents Namespaces
 # ------------------------------------------------------------------
 agentNamespaces:
-- team1
+- kagenti
```

Comment:
This is causing a failure in the CI testing , which is expecting that namespace team1 to exist